scale/docs/rest/node.rst

```diff
@@ -288,41 +288,40 @@ These services provide access to information about the nodes.
 +--------------------+-------------------+--------------------------------------------------------------------------------+
-| hostname           | String            | The full domain-qualified hostname of the node.                                |
-+--------------------+-------------------+--------------------------------------------------------------------------------+
-| port               | Integer           | The port being used by the executor on this node.                              |
-+--------------------+-------------------+--------------------------------------------------------------------------------+
-| pause_reason       | String            | The reason this node is paused if is_paused is true. If is_paused is false     |
-|                    |                   | this field will be set to null. This should provide a brief description        |
-|                    |                   | for user display.                                                              |
+| pause_reason       | String            | (Optional) The reason this node is paused if is_paused is true. If is_paused   |
+|                    |                   | is false, this field will be set to null. This should provide a brief          |
+|                    |                   | description for user display.                                                  |
 +--------------------+-------------------+--------------------------------------------------------------------------------+
-| is_paused          | Boolean           | True if the node is paused and will not accept new jobs for execution.         |
-|                    |                   | Remaining tasks for a previously executing job will complete.                  |
+| is_paused          | Boolean           | (Optional) True if the node is paused and will not accept new jobs             |
+|                    |                   | for execution. Remaining tasks for a previously executing job will complete.   |
 +--------------------+-------------------+--------------------------------------------------------------------------------+
 | **Successful Response**                                                                                                 |
 +--------------------+----------------------------------------------------------------------------------------------------+
-| **Status**         | 201 CREATED                                                                                        |
+| **Status**         | 200 OK                                                                                        |
 +--------------------+----------------------------------------------------------------------------------------------------+
-| **Location**       | URL pointing to the details for the node (should be the same as the request URL)                   |
+| **Location**       | URL pointing to the details for the node (should be the same as the request URL).                  |
```

Comment:
This should be removed. A Location header should only be returned with a 201 response.